website/docs/content/writing-for-ui-copy/index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Writing for UI Copy
```

Comment:
**todo:** titles and headings use sentence case (reference))todo: titles and headings use sentence case ([reference](https://hashicorp.atlassian.net/wiki/spaces/HDS/pages/3342009417/Writing+Style#Capitalization))
```suggestion
title: Writing for UI copy
```